plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -195,18 +195,12 @@ class ServiceDestroyModal extends React.Component {
   }
 
   getDestroyServiceModal() {
-    const { open, service, intl } = this.props;
+    const { open, service } = this.props;
     const serviceName = service.getName();
+    const serviceLabel = this.getServiceLabel();
 
     let itemText = `${StringUtil.capitalize(UserActions.DELETE)}`;
-
-    if (service instanceof Pod) {
-      itemText += " Pod";
-    }
-
-    if (service instanceof ServiceTree) {
-      itemText += " Group";
-    }
+    itemText += ` ${serviceLabel}`;
```

Comment:
Maybe this would be more concise `let itemText = `${StringUtil.capitalize(UserActions.DELETE)} ${serviceLabel}`;`